client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -195,10 +194,8 @@ const Checkout = React.createClass( {
 			return selectedSiteSlug
 				? `/plans/${ selectedSiteSlug }/thank-you`
 				: '/checkout/thank-you/plans';
-		} else if ( cart.create_new_blog && cartItems.hasDomainRegistration( cart ) && ! cartItems.hasPlan( cart ) ) {
-			const domainName = cartItems.getDomainRegistrations( cart )[ 0 ].meta;
-
-			return domainManagementList( domainName );
+		} else if ( cart.create_new_blog && ! cartItems.hasPlan( cart ) ) {
+			return domainManagementList( map( values( receipt.purchases )[ 0 ], 'meta' )[ 0 ] );
```

Comment:
What about using [Lodash `get()`](https://lodash.com/docs#get) here? I suspect we should be able to do the following:

```javascript
const domainName = get( receipt, 'purchases[0].meta[0]' );

return domainManagementList( domainName );
```